### ISSUE TITLE
Error while loading .rmap: Obfuscated maps not supported

### ISSUE THREAD

**Issue by AlbertCoca** [tumic0/GPXSee#506]
Basically, I'm trying to load .rmap format (from twonav). However, I can't due to this error:

Obfuscated maps not supported

Any solution?

Thanks

**Comment by tumic0** [comment]
> Any solution?

Buy the license and use the twonav proprietary app to open the map...

If you already own the license but need to work with the map outside of the twonav proprietary apps then the solution is to not pay money to a company that does not allow you to freely work with data you have payed for the next time...

**Comment by AlbertCoca** [comment]
Thanks, I thought rmap was a supported format for GPXSee and therefore I should be able to open it. however I'm starting to see that the problem is that the map is protected, a pitty that I can't use the map...

**Comment by tumic0** [comment]
> I thought rmap was a supported format for GPXSee and therefore I should be able to open it

Only the non-"protected" (like in all other cases it's just some kind of obfuscation that can be reverse-engineered) rmaps are supported. Today that means only user-created maps as twonav sells even maps, that can be obtained from other sources for free for some small amount of money as "protected" maps. I don't know if that business model works, but there is definitely no community producing some open data rmaps like in case of Garmin (which is also rapidly dying out and all the tools that are available today use a format abandoned by Garmin 20 years ago that is not capable of even display rectangular buildings...).

TLDR - rmaps/twonav is dead.

**Comment by AlbertCoca** [comment]
Thanks so much for the answers, when buying a paper map I can show it to everyone I want and I can put it in any backpack, I just thought buying a digital map would be the same... (So I can open with any open source tool I want since I'm Linux user)

I will drop a call to the map provider (not twonav but a local company) and complain about it, let's see if they manage to offer a more open map format!

Thanks!
